src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
```

Comment:
The class `Circle` should not be directly implementing `Figure` if `Figure` is meant to have methods that are conceptually different, such as `draw()` and `getArea()`. Consider separating these responsibilities into different interfaces or abstract classes as appropriate.